src/wp-admin/includes/dashboard.php

```diff
@@ -1470,7 +1470,8 @@ function cp_dashboard_upgrade() {
 		echo '<p>' . sprintf( __( 'You can continue to use ClassicPress %s at this time. You will be notified when a new version is available that supports future versions of PHP. <a href="%s">Learn more</a> about testing version 2.0.0.' ), $display_version, esc_url( 'https://www.classicpress.net/help-test-classicpress-v2/' ) ) . '</p>';
 	}
 
-	echo '<p>' . __( 'Deprecation notice: Version 1.6 is the last version to support PHP 5.6 - 7.3. The minimum required version for 2.0.0 will be PHP 7.4.' ) . '</p>';
+	echo '<p>' . __( '<strong>Deprecation notice:</strong> Version 1.6 is the last version to support PHP 5.6 - 7.3. The minimum required version for 2.0.0 will be PHP 7.4.' ) . '</p>';
```

Comment:
Can we wrap this in a PHP version check so that we do not have this for servers with PHP 8+?

Reply:
Since 1.5.0 supports PHP 8.0, displaying deprecation notice conditionally might make sense. 

@xxsimoxx with SemVer deprecation notice, if we display it conditionally, would we still comply since some users wouldn't see it? For example, in this case, users on PHP 8+ wouldn't see it.

Reply:
The deprecation can just be in a `CHANGELOG.md` file, but if we follow what we did for 1.6.0 I suggest not to make this conditional.